src/App.js

```diff
@@ -1,11 +1,61 @@
-import Calculator from './components/Calculator';
+import { useEffect, useState } from 'react';
+import { API_URL, HEADERS_LIST } from './api/apiSettings';
+import Calculator from './components/Calculator/Calculator';
+import Quote from './components/Quote/Quote';
 import './assets/styles/App.css';
 
-const App = () => (
-  <>
-    <h1>Welcome to Math Magicians</h1>
-    <Calculator />
-  </>
-);
+const App = () => {
+  const [dailyQuote, setDailyQuote] = useState({ quote: '', author: '' });
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    const fetchQuote = async (quoteType = 'inspirational') => {
+      setIsLoading(true);
+
+      const fetchUrl = `${API_URL}${quoteType}`;
+      const request = new Request(
+        fetchUrl, {
+          method: 'GET',
+          'Content-Type': 'application/json',
+          headers: HEADERS_LIST,
+        },
+      );
+
+      try {
+        const response = await fetch(request);
+        const [json] = await response.json();
+        setDailyQuote((prev) => ({
+          ...prev,
+          ...json,
+        }));
+
+        setHasError(false);
+      } catch (error) {
+        setHasError(true);
+      }
+
+      setIsLoading(false);
+    };
+
+    fetchQuote();
+  }, []);
+
+  return (
+    <>
+      <h1>Welcome to Math Magicians</h1>
+      <Calculator />
+      { isLoading
+        ? (<p>Daily Quote is Loading...</p>)
+        : (
+          <Quote
+            quote={dailyQuote.quote}
+            author={dailyQuote.author}
+          />
+        ) }
+      { hasError && <p>Something went wrong while fetching the daily quote.</p> }
```

Comment:
- [ ] Please do the same thing for these pieces of code too. After completing this you'll have a structure like the following.  🧐️

```
<>
  <h1>Welcome to Math Magicians</h1>
  <Calculator />
  <Quote />
</>
```

Reply:
Thanks again @amejid.
App component renders Calculator and Quote components.